demo/src/screens/componentScreens/HintsScreen.tsx

```diff
@@ -215,6 +223,19 @@ export default class HintsScreen extends Component<HintScreenProps> {
               </View>
             </>
           )}
+
+          <View marginT-100 row center>
+            {targetPosition !== 'flex-start' && <Text marginH-s3>Text pushing button</Text>}
+            <Hint
+              message={'Hint'}
```

Comment:
We should use 'message' so we can text the long massage here as well

Reply:
The whole idea of this use case was to test short message, that's why it's hard coded
I can change it, I just need the short variation of message to be extra short

Reply:
I looked at it again, I prefer to keep it as it is